### PR TITLE
CommandBar, FrameworkElement updates

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
- Displaying a CommandBar fails to render properly, and dynamically added items are not visible.
- `FrameworkElement` may get `Loaded` twice when contained in an `ItemsControl`

## What is the new behavior?
The command bar content shows properly, along with dynamic items, `Loaded` is raised only once.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/Constants.cs [hunk 1]
```diff
@@ -14,6 +14,6 @@ public class Constants
 		public readonly static string AndroidAppName = "uno.platform.unosampleapp";
 
 		// Default active platform when running under Visual Studio test runner
-		public const Platform CurrentPlatform = Platform.Android;
+		public const Platform CurrentPlatform = Platform.Browser;
```
Comment (davidjohnoliver) [inline src/SamplesApp/SamplesApp.UITests/Constants.cs]: not sure if this was intentionally committed... but I approve :p

File: src/Uno.Foundation/Collections/IObservableVector.cs [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
 		/// <summary>
-		/// Occurs when the vector changes.
-		/// </summary>
-		event VectorChangedEventHandler<T> VectorChanged;
-	}
-}
\ No newline at end of file
+        /// Occurs when the vector changes.
+        /// </summary>
+        event VectorChangedEventHandler GenericVectorChanged;
```
Comment (davidjohnoliver) [inline src/Uno.Foundation/Collections/IObservableVector.cs]: should be `UntypedVectorChanged` or `NonGenericVectorChanged` (or just `VectorChangedInternal`)

File: src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBarTemplateSettings.cs [hunk 3]
```diff
@@ -12,14 +12,14 @@ public CommandBarTemplateSettings(CommandBar commandBar)
 		}
 
 		// TODO: Implement
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBarTemplateSettings.cs]: done?

File: src/SamplesApp/SamplesApp.UITests/Constants.cs (hunk 1 above)
Comment (jeromelaban) [reply src/SamplesApp/SamplesApp.UITests/Constants.cs]: It's not intentional, but I think I'll change the way UI Tests are structured to have one per target, so we don't need to do that dance.

File: src/SamplesApp/SamplesApp.Wasm/WasmCSS/Fonts.css [hunk 4]
```diff
@@ -1,4 +1,29 @@
...
...vIhMYF3FxFxf/Ix1AHSMcREQcwDpGA4D8AAPAgP7AQAKAExsQAv7AoBw0Kh4F/j0BwwUeKjQcIDoXcXEXOiAeMwl6egkzHigcREQc8hxtQSD+AAABAIAAAAGAAQAABAAAKQERIREBgP8AAQABAP8AAAACAIAAAAMAAgAABAAJAAApAREhESkBESERAYD/AAEAAYD/AAEAAQD/AAIA/gAAAwCAAAAEgAMAAAQACQAOAAApAREhESkBESERKQERIREBgP8AAQABgP8AAQABgP8AAQABAP8AAgD+AAMA/QAAAAQAgAAABgAEAAAEAAkADgATAAApAREhESkBESERKQERIREpAREhEQGA/wABAAGA/wABAAGA/wABAAGA/wABAAEA/wACAP4AAwD9AAQA/AAABQCAAAAHgAUAAAQACQAOABMAGAAAKQERIREpAREhESkBESERKQERIREpAREhEQGA/wABAAGA/wABAAGA/wABAAGA/wABAAGA/wABAAEA/wACAP4AAwD9AAQA/AAFAPsAAAADA4AAAAgABIAAEwAfACsAAAEyHgIVFA4CIyIuAjU0PgIBFB4CMzI2NwEOAQE+ATU0LgIjIgYHBcB30pxbW5zSd3fSnFtbnNL+t0d5o11Iijr9jSwtAycsLUd5o11IijoEgFuc0nd30pxbW5zSd3fSnFv9wF2jeUctLAJzOor+rDqKSF2jeUctLAABBYAAAAcAAYAAEwAAJRQOAiMiLgI1ND4CMzIeAgcAHjRGKChGNB4eNEYoKEY0HsAoRjQeHjRGKChGNB4eNEYAAgQAAAAHAAMAAA8AIwAAITQ+BDMVIg4EFQEyHgIVFA4CIyIuAjU0PgIEADdkjKvEalikjnVTLgHAKEY0Hh40RigoRjQeHjRGasSrjGQ3gC5TdY6kWAGAHjRGKChGNB4eNEYoKEY0HgADAoAAAAcABIAADwAfADMAACE0PgQzFSIOBBUhNBoBNiwBMxUiBA4CAhUBMh4CFRQOAiMiLgI1ND4CBAA3ZIyrxGpYpI51Uy7+AFKW0gEAASefjf775LuGSQNAKEY0Hh40RigoRjQeHjRGasSrjGQ3gC5TdY6kWJ8BJwEA0pZSgEmGu+T++40BgB40RigoRjQeHjRGKChGNB4ABAIAAQAHgAcAABcAKAA1ADkAAAEhFSEiLgI1ETQ+AjMhARUuASc
```
Comment (dr1rrb) [inline src/SamplesApp/SamplesApp.Wasm/WasmCSS/Fonts.css]: I think there is typo at col 45637

File: src/Uno.Foundation/Collections/IObservableVector.cs [hunk 5]
```diff
@@ -1,13 +1,10 @@
...
 		/// <summary>
-		/// Occurs when the vector changes.
-		/// </summary>
-		event VectorChangedEventHandler<T> VectorChanged;
-	}
-}
\ No newline at end of file
+        /// Occurs when the vector changes.
+        /// </summary>
+        event VectorChangedEventHandler UntypedVectorChanged;
```
Comment (dr1rrb) [inline src/Uno.Foundation/Collections/IObservableVector.cs]: Tabs are better than spaces ! 🙊

File: src/SamplesApp/SamplesApp.Wasm/WasmCSS/Fonts.css (hunk 4 above)
Comment (davidjohnoliver) [reply src/SamplesApp/SamplesApp.Wasm/WasmCSS/Fonts.css]: :D

**Pull request merged**